src/core/ContactUser.cpp

```diff
@@ -327,6 +327,8 @@ void ContactUser::deleteContact()
     if (m_contactRequest) {
         qDebug() << "Cancelling request associated with contact to be deleted";
         m_contactRequest->cancel();
+    }
+    if (m_contactRequest) {
         m_contactRequest->deleteLater();
```

Comment:
applied my proposed changes to my code 1.1.4 release, and could finally delete that pending contact.